kong/runloop/handler.lua

```diff
@@ -1136,7 +1195,7 @@ return {
           }
         end
 
-        if strategy == "off" or kong.configuration.worker_consistency == "strict" then
+        if kong.configuration.worker_consistency == "strict" then
```

Comment:
```lua
get_updated_router = function()
  if kong.db.strategy ~= "off" and kong.configuration.worker_consistency == "strict" then
      local ok, err = rebuild_router(ROUTER_SYNC_OPTS)
```
Do we need to be consistent with the condition?

Reply:
Those options are not used with `dbless`.

Reply:
I added it inside `elseif`